python/hail/representation/genomeref.py

```diff
@@ -236,6 +236,27 @@ def from_file(cls, file):
         """
         return GenomeReference._from_java(Env.hail().variant.GenomeReference.fromFile(Env.hc()._jhc, file))
 
+    @handle_py4j
+    @typecheck_method(file=strlike)
+    def write(self, file):
+        """Write reference genome to file as JSON.
+
+        **Examples**
+
+        >>> my_gr = GenomeReference("new_reference", ["x", "y", "z"], {"x": 500, "y": 300, "z": 200})
+        >>> my_gr.write("output/new_reference.json")
+
+        **Notes**
+
+        Use :py:class:`~hail.GenomeReference.from_file` to reimport the exported reference genome
```

Comment:
by symmetry and convention with other classes, shouldn't `from_file` be changed to `read`?